src/l500/l500-depth.cpp

```diff
@@ -236,7 +236,16 @@ namespace librealsense
 
     void l500_depth_sensor::override_dsm_params( rs2_dsm_params const & dsm_params )
     {
-        algo::depth_to_rgb_calibration::validate_dsm_params( dsm_params );  // throws!
+        try
+        {
+            algo::depth_to_rgb_calibration::validate_dsm_params( dsm_params );  // throws!
+        }
+        catch( invalid_value_exception const & e )
+        {
+            if( ! getenv( "RS2_AC_IGNORE_LIMITERS" ))
+                throw;
```

Comment:
By not-re-translating the exception message a valuable debugging info will be lost

Reply:
That's fine, it's meant to throw, it's only with this special flag that it's supposed to do something else.